README.md

```diff
@@ -61,6 +61,24 @@ Go to http://localhost:8000/admin and create a new user.
 
 ## Access API
 
+### Auth
+
+Go to http://localhost:8000/api/users/token/  
+
+As you can see below, you can fill KEY with username and VALUE with password.  
+After that you should see token, copy that  
+
+Postman
+
+![](https://github.com/zetonteam/zeton_django/blob/auth-with-simplejwt/images/Screenshot_01.png?raw=true)  
```

Comment:
Is super cool, that you paste image. Now it's more understandable.

Reply:
:)